cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterLifecycleTests.java

```diff
@@ -103,6 +105,31 @@ public void testAdapterLifeCycle() throws Exception {
     JsonObject deployedApp = deployedApps.get(0);
     Assert.assertEquals(adapterType, deployedApp.get("id").getAsString());
 
+    List<ScheduleSpecification> schedules = getSchedules(namespaceId, adapterType, AdapterApp.AdapterWorkflow.NAME);
```

Comment:
Note: We don't need to check multiple runs and other workflow related aspects here. We should focus on the lifecyle of adapters. Create, start, stop, delete. Create existing adapters, delete the non-existing adapters. etc.